tests/lib/snaps/test-snapd-policy-app-consumer/meta/snap.yaml

```diff
@@ -83,6 +83,9 @@ apps:
   fwupd:
     command: bin/run
     plugs: [ fwupd ]
+  gpio-memory-control:
+    command: bin/run
+    plugs: [ gpio-memory-control ]
```

Comment:
\o/ missing_interface_spread_test() in run-checks did its job! :)

Reply:
Yes, I actually said "gee, thanks" out loud!

Reply:
Wow, I didn't know we had that. Thank you!!!